pages/sequential.jsx

```diff
@@ -4,6 +4,8 @@ import Header from '../components/header.jsx';
 import Link from '../components/link.jsx';
 import {IntlMixin} from 'react-intl';
 
+var ga = require('react-ga');
```

Comment:
Should I be using `import` like above? This looks like shiny new Javascript I'm not yet familiar with :stuck_out_tongue_closed_eyes: